raptorWeb/authprofiles/views.py

```diff
@@ -0,0 +1,347 @@
+from django.shortcuts import render, redirect
+from django.views.generic import TemplateView
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import HttpResponseRedirect, HttpResponse
+from django.core.exceptions import ObjectDoesNotExist
+from os.path import join
+from logging import getLogger
+
+from raptorWeb import settings
+from authprofiles.forms import UserForm, UserProfileInfoForm, UserLoginForm, DiscordUserInfoForm
+from authprofiles.models import User, UserProfileInfo, DiscordUserInfo
+from raptormc.util import viewContext
+from authprofiles.util import discordAuth
+from raptormc.jobs import player_poller
+
+LOGGER = getLogger('authprofiles.views')
+
+class RegisterUser(TemplateView):
+
+    template_name = join(settings.AUTH_TEMPLATE_DIR, 'registration.html')
+    registered = False
+    register_form = UserForm()
+    extra_form = UserProfileInfoForm()
+
+    def get(self, request):
+
+        dictionary = player_poller.currentPlayers_DB
+        dictionary["registered"] = self.registered
+        dictionary["register_form"] = self.register_form
+        dictionary["extra_form"] = self.extra_form
+        dictionary = viewContext.update_context(context = dictionary)
```

Comment:
`raptormc.util.viewContext` should not be used in this app. The only information added to context from it is Discord Member count information, and that falls outside the scope of authprofiles